docs/UserGuide.md

```diff
@@ -75,20 +75,39 @@ Shows a message explaning how to access the help page.
 Format: `help`
 
 
-### Adding a person: `add`
+### Adding a patient: `addpatient`
 
-Adds a person to the address book.
+Enables the user to add a patient's contact and medical details.
```

Comment:
Dont need "Enables the user to", just "Adds a patient's contact and medical details.